.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 🐞 Bug Report
+about: Report an issue with kpt
+title: ""
+labels:
+  - bug
+assignees: ""
+---
+
+### Expected behavior
+
+### Actual behavior
+
+### Information
+
+<!-- Please include any additional information related to your issue -->
+<!-- EX: Operating System -->
+<!--     Kpt Version (you may find this by running `kpt version` in your shell) -->
```

Comment:
IMO, we just need two pieces of info:

1. Output of `kpt version`. This should include any host info we need for debugging like platform and arch. If something is missing, it's easy to expand it.
2. Minimum reproducible package

Reply:
@runewake2 can you respond here and let's figure out how to bring this to closure 😄

Reply:
@runewake2 ping... lingering work...

Reply:
Should have time to loop back by the end of the week. I've been prioritizing other work. Thanks for the ping.